features/bevragen/persoon/verblijfplaats/adres/onbekendwaardes.feature

```diff
@@ -0,0 +1,108 @@
+# language: nl
+
+@proxy @ready @post-assert
+Functionaliteit: Als gebruiker van de API wil ik geen onbekend waardes ontvangen binnen het adres
+  zodat ik deze niet hoef te (kunnen) interpreteren en ik geen code voor hoef te schrijven om deze situatie af te vangen
+
+  Wanneer in de registratie specifieke waarden gereserveerd zijn voor een onbekende waarde, worden deze waarden niet doorgegeven in de API.
+  Wanneer een element in de registratie een standaardwaarde heeft, die betekent dat de waarde onbekend is, wordt het corresponderende veld niet opgenomen in de response.
+
+Rule: een veld wordt niet opgenomen wanneer het de standaardwaarde bevat
+  Het gaat om de volgende properties en standaardwaardes van de persoon:
+  | property                                         | standaardwaarde  |
+  | ------------------------------------------------ | ---------------- |
+  | verblijfplaats.straat                            | .                |
+  | verblijfplaats.huisnummer                        | 0                |
+  | verblijfplaats.woonplaats                        | .                |
+  | verblijfplaats.nummeraanduidingIdentificatie     | 0000000000000000 |
+  | verblijfplaats.adresseerbaarObjectIdentificatie  | 0000000000000000 |
+
+    Scenario: onbekend waarde "." voor straat 
+    Gegeven de persoon met burgerservicenummer '000000164' heeft de volgende 'verblijfplaats' gegevens
+    | naam                  | waarde |
+    | functie adres (10.10) | W      |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | naam               | waarde |
+    | gemeente_code      | 0518   |
+    | straatnaam (11.10) | .      |
+    | huisnummer (11.20) | 1      |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                              |
+    | type                | RaadpleegMetBurgerservicenummer     |
+    | burgerservicenummer | 000000164                           |
+    | fields              | burgerservicenummer,verblijfplaats  |
```

Comment:
door het vragen van meer fields dan alleen het veld met onbekend/standaard waarde, is het naar mijn mening impliciet geworden dan een veld met standaardwaarde niet wordt geleverd. Ook wordt hierdoor niet gevalideerd/beschreven dat als je een veld vraag met een standaardwaarde dat je toch de correcte verblijfplaats type krijgt

Reply:
Deze heb ik nu omgeschreven. Ik vraag alleen straat en lever dan alleen een verblijfplaats met een type en een leeg verblijfadres. Dit scenario slaagt nu en is explicieter.